### PR TITLE
Psyc Office Overhaul

### PR DESCRIPTION
**Pull request by Readystorm546**
Psyc Office area change and overhaul/referbish.

Brought to you by me and lyis

I used mapmerge don't yell at me jamie

Psyc Office

New psyc office provides amazing gameplay SITTING and LETTING IT OUT. Comes with a bunch of new and amazing feature that I'm sure the player base will love! (Hint: Intercom Location)

Old Psyc Area

This can be changed in future mapping but as for now its just a place holder for where the old psyc office was

# Wiki Documentation

The Wiki Image will have to change along with the location of the psyc office.

### Briefly describe your PR and the impacts of it, in layman's terms. 
Psyc man moves to greener pastures.

### What should players be aware of when it comes to the changes your PR is implementing?
More psyc RP

### What general grouping does this PR fall under? 
Mapping

# Changelog
:cl:  
rscadd: Added new psyc office
rscdel: Removed old psyc office  
/:cl:

### CODE REVIEW

**Comment by MegaEmpirical** [comment]
Location is where the old clerk used to be.

(So it's in-between the commissary and EVA.)

**Comment by patpol4** [comment]
Why? Why is this good for the game?

**Comment by UselessTheremin** [comment]
yeah like this aint super bad but really how does it improve rp or anything? am i missing something?

**Comment by Bibby0110** [comment]
STEP X AND Y DETECTED BURN THIS MAP

**Comment by MegaEmpirical** [comment]
> yeah like this aint super bad but really how does it improve rp or anything? am i missing something?

The psych role is a heavy RP role, giving them greater public access and a nicer room helps with that by making them more accessible and just giving them a little more to work with. The room is designed with the idea of creating different scenarios: we've got the waiting area, added stuff for paperwork, more seats for more people, a couch for someone to lay on, and an irresponsibly placed intercom. 



> Why? Why is this good for the game?

It's nicer, it frees up that spot in medical people have been complaining about being taken up by psych, greater opportunities for psych to interact with the crew, changes up the map a little at no loss, adds a little more life to arrivals and central ring, and creates new opportunities for both crew and antag.

**Review by patpol4 — CHANGES_REQUESTED** [review]
You have to remove all the step_x and step_y variables for this to even pass checks.
Use pixel_x and pixel_y if you want to pixelshift

**Comment by Readystorm546** [comment]
> You have to remove all the step_x and step_y variables for this to even pass checks.
> Use pixel_x and pixel_y if you want to pixelshift

Yep it was 6 in the morning i saw x and y... Ill fix it now

**Comment by alao3alao3** [comment]
maintenance door in the psych office needs to have req_access_txt set to 5

**Comment by alao3alao3** [comment]
also the APC is no longer in a wall
dpng

**Comment by alao3alao3** [comment]
wait what the fuck why does Psych get a request console?

**Comment by Readystorm546** [comment]
> maintenance door in the psych office needs to have req_access_txt set to 5

Ill fix it


> also the APC is no longer in a wall

Ill fix it



> wait what the fuck why does Psych get a request console?

To request meds or call for help

**Comment by alao3alao3** [comment]
Shouldn't the APC be on the outside in maint?

**Comment by patpol4** [comment]
> The psych role is a heavy RP role, giving them greater public access

That spot is literally where we moved clerk from because it was too low traffic

>and a nicer room helps with that by making them more accessible and just giving them a little more to work with.

a nicer room is always good, but does a role based on talking with another player really needs that much room?

>The room is designed with the idea of creating different scenarios: we've got the waiting area, added stuff for paperwork, more seats for more people, a couch for someone to lay on, and an irresponsibly placed intercom. 

all of this either already exists in current psych or can be added easily


> it frees up that spot in medical people have been complaining about being taken up by psych, 

what?

>greater opportunities for psych to interact with the crew, changes up the map a little at no loss, adds a little more life to arrivals and central ring, and creates new opportunities for both crew and antag.

all of this is based on a flawed thought that people will come and talk to the psych suddenly because he has a bigger room that is now in a different location

I don't think this is a good idea

**Comment by MegaEmpirical** [comment]
> That spot is literally where we moved clerk from because it was too low traffic

Currently the arrivals hall has almost nothing going on, moving the clerk made it even more barren. Also, clerk is a role based around interacting with many people in a quick manner. Psych is about working with only a very few and for a longer period, so less traffic is needed. 

> a nicer room is always good, but does a role based on talking with another player really needs that much room?

While it is a decently sized office, it's not an obscene amount of room and the space wasn't being utilized any better than this anyway. Also, part of my consideration was that the psych could now actually have more than one person with them without having to crowd around the equivalent of a closet. 

> all of this either already exists in current psych or can be added easily

A couch would be too much for that tiny room and just because the rest can be done on the original doesn't mean they don't have value here. 

> what?

This was part of Swiss's argument about removing the psych, and other people's as well

> all of this is based on a flawed thought that people will come and talk to the psych suddenly because he has a bigger room that is now in a different location

There is nothing flawed with thinking that changing something up or moving the office from a tiny closet in the back of medical, locked behind department doors, in a busy blood filled department could improve something. 

> I don't think this is a good idea

To each their own.

**Comment by MegaEmpirical** [comment]
Also, what is lost by doing this? One random maintence room that's hallway adjacent?

I don't see any lose to this, only gain.

**Comment by Readystorm546** [comment]
> Shouldn't the APC be on the outside in maint?

I did this as well

**Comment by alao3alao3** [comment]
aight

**Comment by alao3alao3** [comment]
also was that chaise lounge on purpose or is that an incomplete couch

**Comment by patpol4** [comment]
Closing on the basis that I think this doesnt solve the problem at all and just moves a room for no reason.

**Comment by Readystorm546** [comment]
> Closing on the basis that I think this doesnt solve the problem at all and just moves a room for no reason.

The futurist is here, gentlemen! He sees all. He knows what's best for you, whether you like it or not.

**Comment by UselessTheremin** [comment]
wejengin closed readystorms map pr
image

**Comment by MegaEmpirical** [comment]
We lose nothing from this, at least test merge it to see what happens. There's nothing to lose from trying but we could gain.
